lib/hanami/cli/generators/app/action/action.erb

```diff
@@ -1,12 +1,12 @@
-# auto_register: false
 # frozen_string_literal: true
 
-require "<%= underscored_slice_name %>/action"
-
-module <%= classified_slice_name %>
+module <%= classified_app_name %>
   module Actions
 <%= module_controller_declaration %>
-<%= module_controller_offset %>class <%= classified_action_name %> < <%= classified_slice_name %>::Action
+<%= module_controller_offset %>class <%= classified_action_name %> < <%= classified_app_name %>::Action
```

Comment:
(Just a comment, not a request for change!)

That we have to use this `module_controller_offset` as our "spacer.gif" here is kinda hilarious. I've always wished when writing generators that we could just spit out whatever gross looking Ruby and then have it run through a formatter before being saved to disk. Maybe one day!